addons/controllers/testdata/test-cluster-bootstrap.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: run.tanzu.vmware.com/v1alpha3
```

Comment:
The content of this file should be already covered in ```testdata/test-cluster-bootstrap-1.yaml```. can we drop this file then?

Reply:
deleted